chore: rebalance embedded login messaging and add hybrid-use framing (#1327)
auth0/docs-v2: main/docs/authenticate/login/embedded-login.mdx

description: Describes Auth0 Embedded Login used to log users directly into your application, transmitting credentials to the Auth0 server.
title: Embedded Login
---

> Embedded Login allows your users to log in directly within your application, with credentials transmitted to the Auth0 server for authentication. It is a fully supported option for teams that want complete control over the login UI/UX while still using Auth0 as the identity provider.

Reviewer: ```suggestion
Embedded Login allows your users to log in directly within your application and transmit credential to the Auth0 server for authentication. This option is best for teams that want complete control over the login UI/UX while still using Auth0 as the identity provider.
```

> Embedded Login and [Universal Login](/docs/authenticate/login/auth0-universal-login) are not mutually exclusive—you can mix them within the same application. Common patterns include using Universal Login for the initial sign-in, then using embedded flows for [factor enrollment](/docs/authenticate/login/embedded-login/use-cases/enrollment) or step-up authentication. To compare the two and decide which combination fits your use case, read [Hosted Login vs. Embedded Login](/docs/authenticate/login/universal-vs-embedded-login).

Reviewer: ```suggestion
Embedded Login and [Universal Login](/docs/authenticate/login/auth0-universal-login) are not mutually exclusive. For example, you can use Universal Login for initial sign-in and use embedded flows for [factor enrollment](/docs/authenticate/login/embedded-login/use-cases/enrollment) or step-up authentication. To compare the two and decide which combination fits your use case, read [Hosted Login vs. Embedded Login](/docs/authenticate/login/universal-vs-embedded-login).
```
